apps/src/templates/levelSummary/FreeResponseAiSummaryBox.tsx

```diff
@@ -16,6 +19,8 @@ type FreeResponseAiSummaryBoxProps = {
   isPending: boolean;
   studentWorkEvaluations?: StudentWorkEvaluation[];
   evaluationComplete?: boolean;
+  totalStudentCount: number;
+  openDetailedAnalysis?: () => void;
```

Comment:
nit: is this really toggling the details? it both opens and closes them?

Reply:
It will just open them.  Closing the "details" is done through the X on the details...

<img width="702" alt="image" src="https://github.com/user-attachments/assets/ef515d4b-4e84-403d-b4f9-19534730897c" />